example/README.md

```diff
@@ -1,74 +1,81 @@
-localhost:port. If you have a lightstep tracer token, you
-should be able to view the trace data on the lightstep interface. 
-(NOTE: if you wish to use a different OpenTracing tracer, simply replace
-`ls_tracer` with the OpenTracing tracer instance of your choice.)
+Give it few seconds to start and run this in second window:
 
-### Trace a request:
+```
+> python3 client.py   
+```
 
-Navigate to `/request/simple/<int:numrequests>` where numrequests is the number
-of requests you want to send to the page. This will send a request to the server
-app, and the trace will include a span on both the client and server sides. This
-occurs automatically since both the client and server functions are decorated
-with @tracer.trace().
+To see the traced requests, go to Jaeger web interface.
+Select your service name from dropdown on the left (it's
+"jaeger_opentracing_example" in our case) and press Find traces button at the bottom of the page.
 
-Result for `/request/simple/1`:
 
-![request and response spans](https://raw.githubusercontent.com/kcamenzind/flask_opentracing/master/example/img/simple.png)
+![traced request](https://raw.githubusercontent.com/opentracing-contrib/python-flask/example/example/img/jaeger.png)
 
-### Log something to a request:
 
-Navigate to `/log`. This will log a message to the server-side span. The client
-span is created automatically through the @tracer.trace() decorator, while the
-logging occurs within the function by accessing the current span as follows:
+(NOTE: if you wish to use a different OpenTracing tracer instead of Jaeger, simply replace
+`jaeger_tracer` with the OpenTracing tracer instance of your choice.)
 
-```python
-span = tracer.get_span()
-span.log_event("hello world")
-```
-Result for `/log`:
+### Trace a request from browser:
 
-![span with log](https://raw.githubusercontent.com/kcamenzind/flask_opentracing/master/example/img/log.png)
+Browse to http://localhost:5000/log and compare the trace in Jaeger.
+The last one has 2 spans instead of 3. The span of webserver's GET method is missing.
+That is because in client.py we pass already existing span over the wire, whereas the request from your browser has to tracing data in it.
 
 ### Add spans to the trace manually:
 
-Navigate to `/request/childspan/<int:numrequests>`. This will send a request to
-the server app, and the trace will include a span on both the client and server
-sides. The server app will additionally create a child span for the server-side
-span. This example demonstrates how you can trace non-RPC function calls in your
-app, through accessing the current span as follows:
+In log function of the server app, we are creating current_span. This is done to
+trace the work that is being done to render the response to /log. Suppose there's
+a database connection happening. By creating a separate span for it, you'll be able
+to trace the DB request separately from rendering or the response. This gives a
+lot of flexibility to the user.
```

Comment:
maybe it's worth mentioning that `install_patches` will do that automatically for some frameworks like SqlAlchemy, Redis client, etc.